packages/aws-cdk/lib/api/bootstrap/bootstrap-environment.ts

```diff
@@ -183,6 +197,11 @@ export class Bootstrapper {
  */
 const USE_AWS_MANAGED_KEY = 'AWS_MANAGED_KEY';
 
+/**
+ * Magic parameter value that will cause the bootstrap-template.yml to NOT create a CMK but use the default key
```

Comment:
[nitpick] Words like "magic" and pseudo double negatives usually decrease the ease of reading.
How opposed would you be to:
> Predefined value indicating that the default key must be used for bootstrap-template.yml, no CMK will be created.

Reply:
Thanks for the proposal. I replaced my comment.